### PR TITLE
Puppet to master

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: src/katello/client/core/repo.py [hunk 1]
```diff
@@ -140,8 +142,10 @@ def run(self):
         gpgkey   = self.get_option('gpgkey')
         nogpgkey   = self.get_option('nogpgkey')
         unprotected = self.get_option('unprotected')
+        content_type = self.get_option('content_type')
         product = get_product(orgName, prodName, prodLabel, prodId)
-        self.api.create(orgName, product["id"], name, label, url, unprotected, gpgkey, nogpgkey)
+        self.api.create(orgName, product["id"], name, label, url, unprotected,
+                        gpgkey, nogpgkey, content_type)
         print _("Successfully created repository [ %s ]") % name
```
Comment (bbuckingham) [inline src/katello/client/core/repo.py:150]: Will the 'repo info' show the content type information (yum vs puppet)?  If not, we may want to include it.

**Comment by bbuckingham** [comment]
Do we a 'puppet_module' command, similar to what we have today for errata & packages?  If not, we should add something.  This does not have to be addressed by this PR, so we can add tasks to trello if not available.

**Comment by bbuckingham** [comment]
ACK with comments

**Comment by daviddavis** [comment]
@bbuckingham added 2 tasks to Trello. It looks like there is more work to be done on repo info as it displays things like package count.

**Pull request merged**